docs/adr/0001-update-logins-api.md

```diff
@@ -0,0 +1,96 @@
+# Update Logins API
+
+* Status: proposed
+* Date: 2021-05-27
+
+Technical Story: [#4101](https://github.com/mozilla/application-services/issues/4101)
+
+## Context and Problem Statement
+
+The per-field encryption in autofill, which we are planning to replicate in logins, separates the storage and encryption logic by limiting the storage layer to the management of encrypted data.
+
+Applying this approach in logins will break the existing validation and deduping code so we need a way to implement per-field encryption while supporting the validation and de-duping behavior.
+
+
+## Decision Drivers
+
+* Avoiding the implementation of new security approaches that may require additional time and security resources
+* Establishing a standard encyrption approach across components
+
+
+## Considered Options
+
+* Option 1 - Reduce the API functions that require the encryption key and pass the key to the remaining functions
+* Option 2 - Implement a different key management approach
+
+
+## Decision Outcome
+
+(I'm purposely leaving this blank until we arrive at a decision.)
+
+
+## Pros and Cons of the Options
+
+### Option 1 - Reduce the API functions that require the encryption key and pass the key to the remaining functions
+
+* Description
+
+  Currently the below logins API functions would require the per-field encryption key:
```

Comment:
I think we should expand on why the `get` functions need the key, as it's not obvious that it's not because we want to return the decrypted values from our API, but because they call the `validate` and/or `fixup` functions which do need access the the decrypted values to operate correctly. Maybe as a "note" where you mention you are ignoring sync?